A10-avoid-grpclb-and-service-config-for-localhost-and-ip-literals.md

```diff
@@ -0,0 +1,54 @@
+Special case localhost and ip literals for service-config and grpclb-in-DNS
+----
+* Author(s): apolcyn
+* Approver: a11r
+* Status: Draft
+* Implemented in: none currently, suited for all languages
+* Last updated: 2017-12-19
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/Ln6zeUDoo3M
+
+## Abstract
+
+This document discusses a problem and solution to bring down channel
```

Comment:
optional: The problem this doc solves has increased in scope since Java has more fatal errors.  Consider adding some verbiage.

Reply:
yes

Reply:
I've added background on this problem as well - can you please check to see if my description is accurate?